src/functions/read.ts

```diff
@@ -0,0 +1,52 @@
+// Import Node.js Dependencies
+import path from "node:path";
+import { once } from "node:events";
+
+// Import Third-party Dependencies
+import Config from "@slimio/config";
+import TR from "ts-results";
+
+// Import Internal Dependencies
+import schema from "../schema/nodesecurerc.json";
+import { RC, generateDefaultRC } from "../rc";
+import * as CONSTANTS from "../constants";
+
+// CONSTANTS
+const { Ok, Err } = TR;
+
+export interface readOptions {
+  /**
+   * If enabled the file will be created if it does not exist on the disk.
+   *
+   * @default false
+   */
+  createIfDoesNotExist?: boolean;
+}
+
+export async function read(
+  location = process.cwd(),
+  options: readOptions = Object.create(null)
+): Promise<TR.Result<RC, NodeJS.ErrnoException>> {
```

Comment:
Feel free to destructure `Result` too.

Reply:
Not possible, this is a type and ts-results is CJS